doc/website/blog/2023-01-05-atlas-v090.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: "Announcing Atlas v0.9.0: SQL as a First-Class Citizen"
+authors: a8m
+tags: [migration, sql, erd]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+For a long time, one of the most requested feature we've been getting from our users is the ability to
+manage their desired "schema state" using SQL. This is understandable, using Atlas DDL (HCL) can be alien to
+some users, especially those who have never worked with Terraform before. That's why we're excited to announce that
+Atlas v0.9.0 now fully supports SQL.
+
+## Schema as Code (SaC)
+
+Atlas adopts the IaC concept for database schema management. With Atlas, users do not need to plan schema
+changes themselves. Instead of figuring out the correct SQL statements to update their database schemas, users provide
```

Comment:
```suggestion
Atlas applies the common IaC concept of declarative resource management to database schemas. With Atlas, users do not need to plan schema
changes themselves. Instead of figuring out the correct SQL statements to update their database schemas, users provide
```